src/engine/routing_algorithms/many_to_many.cpp

```diff
@@ -71,7 +71,7 @@ inline bool addLoopWeight(const DataFacade<ch::Algorithm> &facade,
     return false;
 }
 
-template <bool DIRECTION>
+template <bool DIRECTION, bool STOP_AT_TOP_LEVEL = false>
```

Comment:
Is there a way that we could avoid having an unused template paramter here? This parameter does not make sense for CH at all

Reply:
yes, the parameter is not relevant for CH. 
I would like to split CH and MLD table implementations, this will allow to improve MLD performance for both `few-to-many` and `many-to-few` queries, because atm only former are improved.